### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mosuka**
Transfer dictionary build commands to each dictionary package and remove dictionary build commands from the CLI package.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,95 +1,187 @@
...
-
-    let (subcommand, some_options) = app.subcommand();
-    let options = some_options.unwrap();
-    let run_cli = match subcommand {
-        "tokenize" => run_tokenize_cli,
-        "build" => run_build_cli,
-        _ => panic!("Subcommand {} is unknown", subcommand),
-    };
+        .arg(
+            Arg::with_name("INPUT_FILE")
```
Comment (johtani) [inline src/main.rs]: INPUT_TEXT instead of INPUT_FILE. name and help.

File: src/main.rs [hunk 2]
```diff
@@ -1,95 +1,187 @@
...
+            // return error message
+            println!("unsupported mode: {}", mode);
+            return;
+        }
+    }
+
+    // output format
+    let output_format = matches.value_of("OUTPUT").unwrap();
+
+    if matches.is_present("INPUT_FILE") {
```
Comment (johtani) [inline src/main.rs]: Change param name to INPUT_TEXT

File: src/main.rs (hunk 1 above)
Comment (mosuka) [reply src/main.rs]: The positional argument `INPUT_FILE` specifies the file name where the text is written, so I don't think it is necessary to change the name. Is there any inconvenience?

Comment (johtani) [reply src/main.rs]: Ah, I confused stdin and file :D
Then, LGTM!

**Pull request merged**